device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/gearbox_config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "sesto-1",
+      "address": "0x1000",
+      "lib_name": "libsai_phy_sesto-1.so",
+      "firmware_path": "/tmp/phy-sesto-1.bin",
```

Comment:
why the firmware path is in tmp?

Reply:
Again, just to re-iterate, the original requirement was to implement on a real switch with real external phys. The project  goals changed to make an initial delivery to support VS. The firmware_path is attribute is still validate. The /tmp designation is a placeholder.